storybookjs/storybook
# Do not try to named import from global

Issue: #14913 

## What I did

Instead of trying to import named exports from the `global` package which don't exist, this imports a default `global` from the package, and then destructures off the pieces that are desired.  This prevents the error experienced in the referenced issue.

I got agreement from @ndelangen for this approach in [discord](https://discord.com/channels/486522875931656193/501692020226654208/846779578332872704) prior to tackling this one.

## How to test

I had a hard time linking my changes into my reproduction.  It seems that snowpack does not play well with the yarn symlinks that are created via `npx sb@next link --local`.  I did, however, copy the built `dist` folder from `client-logger` out of my changed repo and into my reproduction, and the error changed from one usage to another (there are many, and it's hard to copy them all over).  So I'm pretty certain this change does fix the issue.

- Is this testable with Jest or Chromatic screenshots? I don't think so
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
